mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -125,6 +126,8 @@ private[recommendation] trait ALSModelParams extends Params with HasPredictionCo
 
   /** @group expertGetParam */
   def getColdStartStrategy: String = $(coldStartStrategy).toLowerCase(Locale.ROOT)
+
+  setDefault(blockSize -> 4096)
```

Comment:
I just realized that I should set Default of ```blockSize``` in ```ALSModelParams```, so this will apply to both ```ALS``` and ```ALSModel```.